plugins/transformers/superduper_transformers/model.py

```diff
@@ -122,7 +117,7 @@ def compute_metrics(eval_pred):
             return output
 
         def custom_saver():
-            db.replace(model, upsert=True)
```

Comment:
intentional?

Reply:
Yes. The logic of `db.replace` has changed to include `upsert`.